### PR TITLE
sssd: update to 1.15.2

### PR DESCRIPTION
**Pull request by xYuusha**

### CODE REVIEW

**Comment by Vaelatern** [comment]
I dislike the INSTALL.msg, I dislike the diffs that only change space indentation on multi-line variables to tabs, and I dislike the `depends=` lines.
Only after a skim, no actual review.

File: srcpkgs/sssd/template [hunk 1]
```diff
@@ -1,24 +1,38 @@
...
+	--without-semanage
+	--disable-cifs-idmap-plugin
+	--without-samba"
 hostmakedepends="pkg-config nscd bind xmlcatmgr docbook-xsl"
 makedepends="pam-devel popt-devel talloc-devel tdb-devel tevent-devel ldb-devel
- ding-libs-devel libldap-devel mit-krb5-devel c-ares-devel glib-devel
- libxslt-devel python-devel nss-devel libnfsidmap-devel"
+	ding-libs-devel libldap-devel mit-krb5-devel c-ares-devel glib-devel
+	libxslt-devel python-devel python3-devel nss-devel libnfsidmap-devel http-parser-devel jansson-devel"
+depends="tevent"
```
Comment (the-maldridge) [inline srcpkgs/sssd/template]: This should be resolved automatically.

File: srcpkgs/sssd/template [hunk 2]
```diff
@@ -1,24 +1,38 @@
...
- --without-python3-bindings"
+configure_args="--without-selinux
+	--without-semanage
+	--disable-cifs-idmap-plugin
+	--without-samba"
 hostmakedepends="pkg-config nscd bind xmlcatmgr docbook-xsl"
 makedepends="pam-devel popt-devel talloc-devel tdb-devel tevent-devel ldb-devel
- ding-libs-devel libldap-devel mit-krb5-devel c-ares-devel glib-devel
- libxslt-devel python-devel nss-devel libnfsidmap-devel"
+	ding-libs-devel libldap-devel mit-krb5-devel c-ares-devel glib-devel
```
Comment (the-maldridge) [inline srcpkgs/sssd/template]: Our style is minimal indents, please change this back.

File: srcpkgs/sssd/template [hunk 3]
```diff
@@ -1,24 +1,38 @@
 # Template file for 'sssd'
 pkgname=sssd
-version=1.13.4
+version=1.15.2
 revision=1
 build_style=gnu-configure
-configure_args="--with-selinux=no --with-semanage=no
- --disable-cifs-idmap-plugin --without-samba --disable-config-lib
- --without-python3-bindings"
+configure_args="--without-selinux
```
Comment (the-maldridge) [inline srcpkgs/sssd/template]: no tab indent

File: srcpkgs/sssd/template [hunk 4]
```diff
@@ -1,24 +1,38 @@
...
+	--without-semanage
+	--disable-cifs-idmap-plugin
```
Comment (the-maldridge) [inline srcpkgs/sssd/template]: pretty sure this breaks samba support if you do this

File: srcpkgs/sssd/template [hunk 5]
```diff
@@ -1,24 +1,38 @@
...
+	/var/lib/sss/pipes/private 0700 root root
+	/var/lib/sss/mc 0700 root root
+	/var/lib/sss/keytabs 0700 root root
+	/var/lib/sss/gpo_cache 0700 root root
+	/var/lib/sss/db 0700 root root
+	/etc/sssd/ 0700 root root
+	/var/sssd/conf.d 0700 root root"
 short_desc="System Security Services Daemon"
-maintainer="Michael Aldridge <aldridge.mac@gmail.com>"
+maintainer="Yuusha Spacewolf <xyuusha@paranoici.org>"
```
Comment (the-maldridge) [inline srcpkgs/sssd/template]: If you're pulling ownership for this package you will also need to pull ownership for the parts of the depchain of this package that I currently hold. (ding libs, tevent, tdb, ldb, and I think one other).

**Comment by xYuusha** [comment]
Sorry about the INSTALL.msg. Pushed wrong commit.

About the indentation and spacing, is there a style rule to use? Probably changed to tabs without noticing.

File: srcpkgs/sssd/files/sssd/run [hunk 6]
```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+exec chpst sssd -i
```
Comment (the-maldridge) [inline srcpkgs/sssd/files/sssd/run]: calling chpst with no arguments doesn't do anything, perhaps you should either set some arguments (if sssd can run without root privs) or remove the chpst line if it needs to run as root as it doesn't do anything at the moment.

File: srcpkgs/sssd/template (hunk 4 above)
Comment (xYuusha) [inline srcpkgs/sssd/template]: What is the part that i changed that brakes samba?

Comment (the-maldridge) [inline srcpkgs/sssd/template]: `--disable-cifs-idmap-plugin`

Comment (xYuusha) [inline srcpkgs/sssd/template]: The original template already had that line.
Do you think is a good idea to remove it? It already had the '--without-samba' flag.

Comment (the-maldridge) [inline srcpkgs/sssd/template]: No go ahead and leave it, this is harder to get working if you throw samba into the mix.

File: srcpkgs/sssd/template (hunk 1 above)
Comment (xYuusha) [inline srcpkgs/sssd/template]: Are you talking about the tevent depends?

When i install it in a clean environment, libtevent is missing. Do i need to add something else to have the dependency resolved automatically?

**Comment by xYuusha** [comment]
@the-maldridge Last changes we discussed are pushed now.

**Comment by the-maldridge** [comment]
Have fun!

**Pull request closed without merging**